apps/mobile/src/components/ContextMenu/ContextMenu.js

```diff
@@ -62,23 +56,22 @@ export default function ContextMenu () {
   )
 }
 
-function ContextMenuItem ({ widget, groupSlug, rootPath, group }) {
+function MenuItem ({ widget, groupSlug, rootPath, group }) {
   const { t } = useTranslation()
   const { listItems, loading } = useContextWidgetChildren({ widget, groupSlug })
   const logout = useLogout()
   const hasResponsibility = useHasResponsibility({ forCurrentGroup: true, forCurrentUser: true })
   const canAdmin = hasResponsibility(RESP_ADMINISTRATION)
   const [{ currentGroup }] = useCurrentGroup()
 
-  const title = widget.title
+  const title = t(widget.title)
```

Comment:
Need to bookmark this

Reply:
Ok, so we are now bringing useTranslate into the packages. And otherwise we've been resolving the title in the presenters. Why now are we bringing t back here?

Reply:
I am glad you asked, this was done because in the current presentations of widget these titles don't get translated and this was a force. I was gonna ask you about this, but for the moment this renders the expect UI.